### PR TITLE
Tony/aws upload

### PR DESCRIPTION
**Pull request by Tony1023**

### CODE REVIEW

**Comment by Tony1023** [comment]
Sample request:
```
curl --location --request PUT 'http://localhost:3000/api/assets/temp/image.jpeg' \
--header 'Content-Type: image/jpeg' \
--data-binary '<your-jpeg-file>'
```

File: controller.js [hunk 1]
```diff
@@ -678,22 +668,33 @@ const getAllMenus = (req, res) => {
...
+    })
+    .catch((err) => {
+      console.log(err);
+      res.status(err.statusCode).send({
+        message: err.message
+      });
+    });
 };
 
+const uploadAsset = async (req, res) => {
```
Comment (cchyung) [inline controller.js:685]: Can we make separate methods for uploading a restaurant image vs. a dish image, etc.?  

That way the frontend does't have to deal with resolving a path, the backend can handle that instead based on passed in restaurantID and dishID

Comment (cchyung) [reply controller.js:685]: same for fetching assets.

Comment (Tony1023) [reply controller.js]: Let's do this in another pull request. I think there will be another load of changes coming in for this. Let's get this one merged first - I'm branching the new PR off of this one.

**Pull request merged**